cmake/utils.cmake

```diff
@@ -413,6 +413,13 @@ function (define_gpu_extension_target GPU_MOD_NAME)
   target_compile_options(${GPU_MOD_NAME} PRIVATE
     $<$<COMPILE_LANGUAGE:${GPU_LANGUAGE}>:${GPU_COMPILE_FLAGS}>)
 
+  if (TORCH_CXX_FLAGS)
+      # keep the same flags as torch, especially for flags like
+      # GLIBCXX_USE_CXX11_ABI
+      message(STATUS "Find and use torch CXX flags: ${TORCH_CXX_FLAGS}")
+      target_compile_options(${GPU_MOD_NAME} PRIVATE ${TORCH_CXX_FLAGS})
```

Comment:
Does this override any existing flags or append the pytorch flags?

Reply:
see https://cmake.org/cmake/help/latest/command/target_compile_options.html#target-compile-options , I think it appends flags rather than overriding them.